tests/pass-dep/shims/mmap.rs

```diff
@@ -163,7 +172,9 @@ fn test_mremap() {
 }
 
 fn main() {
-    test_mmap();
+    test_mmap(libc::mmap);
+    #[cfg(target_os = "linux")]
+    test_mmap(libc::mmap64);
```

Comment:
You implemented this shim for all unices, but you are testing it only for Linux -- why?